scripts/windows/build/Publish-Branch.ps1

```diff
@@ -227,6 +235,7 @@ else {
  # Publish IoTEdgeQuickstart and LeafDevice
  #>
 
+ # Subject to remove once other pipelines are updated to use the new path
```

Comment:
I don't understand this comment...?

Reply:
If I remove this section, no files will be published to smoke/IotEdgeQuickStart folder (currently used in other build pipelines); therefore it will break the build.  However I can't update build pipeline right now until my change is committed to master.  Therefore I leave this comment here; once this PR is committed, I will update the build pipelines and then remove this block.